### PR TITLE
Update cloudforms.adoc

### PR DESCRIPTION
**Pull request by shraderdm**
Added steps for CloudForms 4.1 and a quick note that configuring a router on the master is no longer necessary.

### CODE REVIEW

**Comment by shraderdm** [comment]
Added additional info for CloudForms 4.1 integration.

File: playbooks/operationalizing/cloudforms.adoc [hunk 1]
```diff
@@ -186,11 +186,32 @@ NOTE: If an all in one deployment of CloudForms is being utilized, all roles can
 
 === Configure the OpenShift CloudForms Provider
 
+CloudForms 4.0:
```
Comment (etsauer) [inline playbooks/operationalizing/cloudforms.adoc]: Can we make this a subheading? i.e.
`==== CloudForms 4.0`

File: playbooks/operationalizing/cloudforms.adoc [hunk 2]
```diff
@@ -186,11 +186,32 @@ NOTE: If an all in one deployment of CloudForms is being utilized, all roles can
...
 In the CloudForms web console, select the *Containers* tab and then select *Providers*.
 
-Select the *Configuration* button and then *Add a New Containers Provider*
+Select the *Configuration* button and then *Add a New Containers Provider*.
+
+Enter a name for the provider, select *OpenShift Enterprise* in the type, enter the address of the master and then the token retrieved earlier into the *Token* field. Click *Validate* to confirm connectivity. 
+
+Click *Add* to add the container provider.
+
+CloudForms 4.1:
```
Comment (etsauer) [inline playbooks/operationalizing/cloudforms.adoc]: same as above

**Comment by etsauer** [comment]
@shraderdm just a couple of formatting comments. Otherwise looks good!

File: playbooks/operationalizing/cloudforms.adoc (hunk 1 above)
Comment (shraderdm) [inline playbooks/operationalizing/cloudforms.adoc]: Done. :)

File: playbooks/operationalizing/cloudforms.adoc (hunk 2 above)
Comment (shraderdm) [inline playbooks/operationalizing/cloudforms.adoc]: Done here as well. :)

**Comment by etsauer** [comment]
great! thanks.

**Pull request merged**